app/components/gutenberg/gravity-forms/Form/AntForm.tsx

```diff
@@ -0,0 +1,42 @@
+// React dependencies
+import React, { RefObject } from 'react';
+
+// Antd dependenies
+import { Form } from 'antd';
+import type { FormInstance } from 'antd/es/form';
+
+// Component dependencies
+import { Fields } from './Fields/Fields';
+
+// Types
+interface AntFormProps  {
+  formData?: any;
```

Comment:
@Danishkhurshid Let's add a TODO comment to all `any`s to make sure we come back to them later and type correctly